packages/pluggableWidgets/slider-web/src/utils/useOnChangeDebounced.ts

```diff
@@ -0,0 +1,21 @@
+import { Big } from "big.js";
+import { useCallback, useMemo, useRef } from "react";
+import { debounce, executeAction } from "@mendix/piw-utils-internal";
+import { SliderContainerProps } from "../../typings/SliderProps";
+
+export function useOnChangeDebounced(props: SliderContainerProps) {
+    const valueObjRef = useRef(props.valueAttribute);
+    valueObjRef.current = props.valueAttribute;
```

Comment:
This seems strange, this means that on every call of this hook, the `valueObjRef` changes. This sounds especially bad for a ref. Why do we need this?